CVEs/CVE-2021-23364.json

```
@@ -0,0 +1,58 @@
+{
+    "CVE": "CVE-2021-23364",
+    "state": "PUBLISHED",
+    "repository": "https://github.com/browserslist/browserslist.git",
+    "prePatch": {
+        "commit": "209adf9e0051fa39a2b25354cffd493300f34b02",
+        "weaknesses": [
+            {
+                "location": {
+                    "file": "index.js",
+                    "line": 802
+                },
+                "explanation": "The package browserslist from 4.0.0 and before 4.16.5 are vulnerable to Regular Expression Denial of Service (ReDoS) during parsing of queries."
+            },
+            {
+                "location": {
+                    "file": "index.js",
+                    "line": 827
+                },
+                "explanation": "The package browserslist from 4.0.0 and before 4.16.5 are vulnerable to Regular Expression Denial of Service (ReDoS) during parsing of queries."
+            },
+            {
+                "location": {
+                    "file": "index.js",
+                    "line": 855
+                },
+                "explanation": "The package browserslist from 4.0.0 and before 4.16.5 are vulnerable to Regular Expression Denial of Service (ReDoS) during parsing of queries."
+            },
+            {
+                "location": {
+                    "file": "index.js",
+                    "line": 890
+                },
+                "explanation": "The package browserslist from 4.0.0 and before 4.16.5 are vulnerable to Regular Expression Denial of Service (ReDoS) during parsing of queries."
+            },
+            {
+                "location": {
+                    "file": "index.js",
+                    "line": 921
+                },
+                "explanation": "The package browserslist from 4.0.0 and before 4.16.5 are vulnerable to Regular Expression Denial of Service (ReDoS) during parsing of queries."
+            },
+            {
+                "location": {
+                    "file": "index.js",
+                    "line": 925
+                },
+                "explanation": "The package browserslist from 4.0.0 and before 4.16.5 are vulnerable to Regular Expression Denial of Service (ReDoS) during parsing of queries."
+            },
```

```suggestion
            }
```

This superfluous comma is causing the JSON to not parse.